lib/OpenQA/Scheduler/Scheduler.pm

```diff
@@ -574,7 +574,7 @@ sub job_grab {
     $worker->set_property('JOBTOKEN', $token);
     $job_hashref->{settings}->{JOBTOKEN} = $token;
 
-    my $updated_settings = $job->assets_from_settings();
+    my $updated_settings = $job->register_assets_from_settings();
```

Comment:
that we call functions sometimes with () and sometimes without is the next policy for perlcritic :)